### PR TITLE
add `packages/orangutan`

### PR DESCRIPTION
**Pull request by nickhsine**
Follow [our conclusions](https://github.com/twreporter/orangutan/pull/6#issuecomment-603169015), I simply created `packages/orangutan` sub-package.
This sub-package would be used to build `@twreporter/orangutan` npm packge.

## Follow Up Action
Since `@twreporter/orangutan` uses `@twreporter/dual-channel`, `@twreporter/scrollable-image` and `@twreporter/timeline`, there is API to follow.

In each sub-package's main file (usually `src/index.js`), 
we need to expose two functions, 
`getWebpackEntry` and
`getTestData`
You can see the following commits
https://github.com/twreporter/orangutan-monorepo/pull/6/commits/1444186729b34defa994664d1b45de72ab95a9c5
https://github.com/twreporter/orangutan-monorepo/pull/6/commits/ca8ef3dbebde5a4ea4004ec6f9f3f5a0e407d23c
 for further understanding.

`getWebpackEntry returns {string}`,  that string tells `@twreporter/orangutan` to use it as webpack entry point.

`getTestData returns {*}`, that returned value would be used to pass into your `buildEmbeddedCode` as param. Hence, you need to return the data which `buildEmbeddedCode` knows about.

### For dual-channel and scrollable-image
The root makefile does not `build-dist` (build webpack bundles) anymore.
Thus, we have to delete `build-root-dist` and `copy-root-dist` targets from our makefile (package level).
In the future, we need to maintain package level `webpack.config.js`, and build/serve webpack bundles by ourselves.

### CODE REVIEW

File: packages/orangutan/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+        },
+      },
+      {
+        test: /\.(jpe?g|png|gif|mp4)$/i,
+        use: {
+          loader: 'url-loader',
+        },
+      },
+      {
+        test: /\.svg$/,
```
Comment (YuCJ) [inline packages/orangutan/webpack.config.js]: Are the svg files imported in `.js` already handled by `babel-plugin-inline-react-svg` in `babel-loader`? Is here we intended to use both `babel-plugin-inline-react-svg` and `@svgr/webpack`?

File: packages/orangutan/webpack.config.js [hunk 2]
```diff
@@ -88,14 +88,19 @@ BundleListPlugin.prototype.apply = function(compiler) {
       const targetPackage = packagesList.find(function(element) {
         return filename.indexOf(`${element}`) !== -1
       })
+
+      const scriptSrc = isProduction
+        ? `${cdnLinkPrefix}/${filename}`
+        : `/dist/${filename}`
```
Comment (taylrj) [inline packages/orangutan/webpack.config.js]: `/dist/`  -> `distDir` ?

File: makefile [hunk 3]
```diff
@@ -9,19 +9,9 @@ dev: check-dep
...
-build-lib: check-dep
+build: check-dep
 	@echo "$(P) Run \`npm run build\` of all packages with \`scripts.build\`"
 	$(BIN_DIR)/lerna run --stream build
-	NODE_ENV=production $(BIN_DIR)/babel packages/index.js --out-dir lib --root-mode upward
-
-build-dist:
-	$(BIN_DIR)/webpack --config webpack.config.js
-
-clean:
```
Comment (taylrj) [inline makefile]: Remove `clean` from

`.PHONY: check-dep dev build clean changed-packages-unit-test integration-test test prettier lint dep-intersect`

?

**Comment by nickhsine** [comment]
Reminder for you guys:
Because of this commit https://github.com/twreporter/orangutan-monorepo/pull/7/commits/1e944529f1daf80db37dc8a2b2e2c3760daba656 , in theory, we don't have to `import 'regenerator-runtime/runtime' ` or `import @babel/polyfill` at the top of entry point.

However, we need to add `@babel/polyfill` to `package.json#dependencies`.

File: packages/orangutan/webpack.config.js (hunk 1 above)
Comment (nickhsine) [reply packages/orangutan/webpack.config.js]: This one is for `@twreporter/orangutan`.
However, in `packages/orangutan`, we don't have any `.`jsx` files.
I could remove it for now.
If we need it, we can add it back.

File: packages/orangutan/webpack.config.js (hunk 2 above)
Comment (nickhsine) [reply packages/orangutan/webpack.config.js]: `distDir = './dist'` which doesn't equal to `/dist`

**Pull request merged**